clients/admin-ui/cypress/e2e/nav-bar.cy.ts

```diff
@@ -1,33 +1,25 @@
-// TODO: Update Cypress test to reflect the nav bar 2.0
-describe.skip("Nav Bar", () => {
+describe("Nav Bar", () => {
   beforeEach(() => {
     cy.login();
   });
 
   it("Renders all page links", () => {
     cy.visit("/");
 
-    cy.getByTestId("nav-link-Privacy Requests");
-    cy.getByTestId("nav-link-Connections");
-    cy.getByTestId("nav-link-User Management");
-    cy.getByTestId("nav-link-Datasets");
-    cy.getByTestId("nav-link-Taxonomy");
-    cy.getByTestId("nav-link-Systems");
-  });
-
-  it("Renders the active page based on the current route", () => {
-    // Start on the dataset page.
-    cy.visit("/dataset");
+    cy.getByTestId("nav-link-home")
+      .should("have.attr", "href", "/")
+      .contains("Home");
 
-    // The nav should reflect the active page.
-    cy.getByTestId("nav-link-Datasets").should("have.attr", "data-active");
-    cy.getByTestId("nav-link-Taxonomy").should("not.have.attr", "data-active");
+    cy.getByTestId("nav-link-privacy-requests")
+      .should("have.attr", "href", "/privacy-requests")
+      .contains("Privacy requests");
 
-    // Navigate by clicking a nav link.
-    cy.getByTestId("nav-link-Taxonomy").click();
+    cy.getByTestId("nav-link-data-map")
+      .should("have.attr", "href", "/system")
+      .contains("Data map");
 
-    // The nav should update which page is active.
-    cy.getByTestId("nav-link-Taxonomy").should("have.attr", "data-active");
-    cy.getByTestId("nav-link-Datasets").should("not.have.attr", "data-active");
+    cy.getByTestId("nav-link-management")
+      .should("have.attr", "href", "/taxonomy")
+      .contains("Management");
```

Comment:
is it possible to restore some of the tests that check for "active" as well?

also, should we add tests for the side nav, or is that a separate chunk of work?